### PR TITLE
Added vue.js course pt_BR

### PR DESCRIPTION
**Pull request by VictorSega**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Description 
Basic Vue.js course for beginners.

### Why is this valuable (or not)?
It's simple and good for beginners.

### How do we know it's really free?
Need only to enter on site and watch course.

### Checklist:
- [X] Not a duplicate
- [X] Included author(s) if appropriate
- [X] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-pt_BR.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 * [Ionic](#ionic)
 * [Java](#java)
 * [JavaScript](#javascript)
+* [Vue.js](#vue.js)
```
Comment (SethiPandi) [inline free-courses-pt_BR.md:19]: Can you remove the dot from the URL. ie `#vuejs`.

Also please slot Vue.js in alphabetical order in the table of contents.

File: free-courses-pt_BR.md [hunk 2]
```diff
@@ -130,6 +131,11 @@
 * [Fast & Furious](https://www.youtube.com/playlist?list=PLy5T05I_eQYOoUz2TtAqq35RLCc-xBZCe) - Codecasts (YouTube)
 
 
+### Vue.js
```
Comment (SethiPandi) [inline free-courses-pt_BR.md:134]: Could you please slot this category in alphabetical order in line with the rest of this document.

**Review by SethiPandi — CHANGES_REQUESTED** [review]
I've left you some comments!

You can get more guidance on the formatting and conventions in this repository by checking out the contribution guide (CONTRIBUTING.md).

You can also click on the failed checks below to get more information on any check failures. You'll be able to see exactly what was wrong and how to fix it!

**Comment by VictorSega** [comment]
Thanks!

I Fixed this things :)

**Pull request merged**